addon/sidebar.js

```diff
@@ -1,26 +1,9 @@
-
-
-function sendEvent(args) {
-  // We bucket to the nearest 50px:
-  args.cd1 = Math.round(window.innerWidth / 50) * 50;
-  args.type = "sendEvent";
-  browser.runtime.sendMessage(args);
-}
-
-function element(selector) {
-  return document.querySelector(selector);
-}
-
-element(".feedback-button").addEventListener("click", () => {
```

Comment:
We actually still have the feedback button in the onboarding page, but I'm going to go ahead and merge this now since it is not hooked up to anything. When it is hooked up we will want to put this back, or maybe remove the feedback button from the onboarding page entirely.